core/services/ocr2/plugins/ocr2keeper/evm/mercury_lookup.go

```diff
@@ -0,0 +1,343 @@
+package evm
+
+import (
+	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/avast/retry-go/v4"
```

Comment:
is this library vetted?

Reply:
no. do we just tag security to ask about this?

Reply:
created security ticket to vet this and the cache libraries:
https://smartcontract-it.atlassian.net/servicedesk/customer/portal/4/SECHD-6620